MonoGame.Framework/Audio/NoMicrophoneConnectedException.cs

```diff
@@ -0,0 +1,36 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.Serialization;
+
+namespace Microsoft.Xna.Framework.Audio
+{
+
+    /// <summary>
+    /// The exception thrown when no audio hardware is present, or driver issues are detected.
+    /// </summary>
+    [DataContract]
+#if WINRT
+    public sealed class NoMicrophoneConnectedException : Exception
+#else
+    public sealed class NoMicrophoneConnectedException : ExternalException
```

Comment:
I wrongly copied/pasted it from```NoAudioHardwareException``` (which is an ```ExternalException```).
I'm updating this PR (let me know if you'd like me to squash and resubmit).